s3-proxy/nginx.conf

```diff
@@ -57,7 +57,10 @@ http {
             }
 
             # Proxy everything else to S3.
-            proxy_pass 'https://$s3_bucket.s3.amazonaws.com/$s3_path$is_args$args';
+            # Use $request_uri rather than $s3_path because it needs to stay encoded.
+            if ($request_uri ~ "^/[^/]+/(.*)") {
```

Comment:
so the scheme has already been stripped from `$requiest_uri`? I am curious if we will ever get something other than s3:// paths and if not then why do we need to handle multiple leading `/`?

Reply:
`$request_uri` contains just the path, starting with `/`, without a scheme. But, it contains the bucket name first - so `/S3_BUCKET/S3_PATH`